src/Config/ObjectModel/ApiType.cs

```diff
@@ -9,5 +9,7 @@ namespace Azure.DataApiBuilder.Config.ObjectModel;
 public enum ApiType
 {
     REST,
-    GraphQL
+    GraphQL,
+    // This is required to indicate features common between REST/GraphQL APIs.
+    Both
```

Comment:
Looks like apiType `both` is really only used to print out error messages.
in RuntimeConfigValidator:
e.g.:
"{apiType} {apiProperty} cannot be null or empty."
The apiType name `Both` seems inappropriate for the error message.

In future, if there are other APIs, `Both` will be restrictive. Perhaps, `All` makes more sense?